generators/connection/templates/sequelize.js

```diff
@@ -6,6 +6,22 @@ module.exports = function() {
   const app = this;
   const connectionString = app.get('sequelize');
   const sequelize = new Sequelize(connectionString);
+  const oldSetup = app.setup;
 
   app.set('sequelizeClient', sequelize);
+
+  app.setup = function(...args) {
+    // Set up data relationships
```

Comment:
I would call `const result = oldSetup.apply(this, args);` first and then `return result;` in the end just in case.